src/test/simple_gadget_test.cpp

```diff
@@ -0,0 +1,270 @@
+#include "circuits/simple_gadgets.hpp"
+#include "snarks_alias.hpp"
+
+#include "gtest/gtest.h"
+#include <libff/common/default_types/ec_pp.hpp>
+
+// Access the `from_bits` function and other utils
+#include "circuits/circuits-util.hpp"
+#include "util.hpp"
+
+using namespace libsnark;
```

Comment:
Remove this `using` directive. Let's avoid to use such directives (Moreover, the namespace prefix seems to be used in the tests body). `using namespace libzeth` is ok as we test functionalities of this library here.